library/src/main/java/com/datatorrent/lib/io/block/BlockMetadata.java

```diff
@@ -227,37 +226,10 @@ public String getFilePath()
       return filePath;
     }
 
-    /**
-     * Get if blocks should be read in sequence
-     * @return readBlockInSequence
-     */
-    public boolean isReadBlockInSequence()
```

Comment:
Why are we removing this functionality? I remember that you mentioned javadoc is not clear, so we need to improve that. But why remove the feature?